src/formulate.backoffice.ui/styles/lib/layout-designer.scss

```diff
@@ -78,6 +82,7 @@
     font-weight: bold;
     font-size: 18px;
     transition: background-color 0.25s;
+    text-decoration:underline;
```

Comment:
Please remove this underline. If you wish to retain the underline, submit a separate issue explaining the reasoning behind it, and we can discuss it there. Would like to restrict this pull request to the issue it addresses.